server/views/pages/choosePrisoner.njk

```diff
@@ -1,12 +1,27 @@
 {% extends "../partials/layout.njk" %}
-{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from  "../partials/incomingMovementCardMacro.njk" import incomingMovementCard %}
 {% set pageTitle = applicationName + " - Home" %}
 {% block content %}
-    <main class="app-container govuk-body">
-        <h1>Select prisoner to add to the establishment roll</h1>
-        <h2>Prisoners arriving from court</h2>
+    <div class="govuk-!-margin-bottom-8">
+        <div class="govuk-width-container govuk-body">
+            <h1>Select prisoner to add to the establishment roll</h1>
+                <h2 class="govuk-heading-m govuk-!-margin-top-9">Prisoners arriving from court</h2>
+                    {% for key, movements in incomingMovements %}
```

Comment:
Could you avoid iterating/filtering by getting the appropriate array directly? 
something like? 
`{% for movements in incomingMovements.get('fromCourt') %}`